assets/js/src/core/modules/element/actions/open/open.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import React from 'react'
+import { type ItemType } from '@Pimcore/components/dropdown/dropdown'
+import { Icon } from '@Pimcore/components/icon/icon'
+import { checkElementPermission } from '@Pimcore/modules/element/permissions/permission-helper'
+import { useTranslation } from 'react-i18next'
+import { type Element } from '@Pimcore/modules/element/element-helper'
+import { useElementHelper } from '@Pimcore/modules/element/hooks/use-element-helper'
+import { type ElementType } from 'types/element-type.d'
+
+export interface UseOpenHookReturn {
+  openContextMenuItem: (node: Element, onFinish?: () => void) => ItemType
+}
+
+export const useOpen = (elementType: ElementType): UseOpenHookReturn => {
+  const { t } = useTranslation()
+  const { openElement } = useElementHelper()
+
+  const openContextMenuItem = (node: Element): ItemType => {
+    return {
+      label: t('element.open'),
+      key: 'open',
+      icon: <Icon value={ 'union' } />,
+      hidden: !checkElementPermission(node.permissions!, 'rename') || node.isLocked,
```

Comment:
The permission check needs to be based on the `view` permission and we need to remove the `|| node.isLocked` part of the condition.

Reply:
absolutely correct - corrected it